test/test_helper.rb

```diff
@@ -2,14 +2,12 @@
 require_relative '../config/environment'
 require 'rails/test_help'
 
-module ActiveSupport
-  class TestCase
-    # Run tests in parallel with specified workers
-    parallelize(workers: :number_of_processors, with: :threads)
+class ActiveSupport::TestCase
+  # Run tests in parallel with specified workers
+  parallelize(workers: :number_of_processors)
 
-    # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
-    fixtures :all
```

Comment:
- [ ] There is a failing test on my end, if this is the case kindly fix this. I can see that if I use `rspec spec` command, the tests dont show, however when I run `bundle exec rspec spec`, the tests run with one failing 


Screenshot
![image](https://github.com/Ademola-coding/Recipe_app/assets/81932478/44daf9d7-48d2-4bde-9aef-dddbec020acb)